cluster/gce/configure-vm.sh

```diff
@@ -452,10 +452,11 @@ network_policy_provider: '$(echo "$NETWORK_POLICY_PROVIDER" | sed -e "s/'/''/g")
 enable_manifest_url: '$(echo "${ENABLE_MANIFEST_URL:-}" | sed -e "s/'/''/g")'
 manifest_url: '$(echo "${MANIFEST_URL:-}" | sed -e "s/'/''/g")'
 manifest_url_header: '$(echo "${MANIFEST_URL_HEADER:-}" | sed -e "s/'/''/g")'
-master_name: '$(echo "${MASTER_NAME:-}" | sed -e "s/'/''/g")'
+master_name: '$(echo "${KUBERNETES_MASTER_NAME:-}" | sed -e "s/'/''/g")'
```

Comment:
There was an error. Please not that we set KUBERNETES_MASTER_NAME in [common.sh](https://github.com/kubernetes/kubernetes/blob/master/cluster/common.sh#L544). In effect, variable `master_name` had incorrect value in salt config (was empty).